scripts/cloudbuild/build.py

```diff
@@ -125,8 +133,10 @@ def generate_body(template_target, name, branch, options, remoteCacheStrategy):
         print(f"New target found")
         # Create new target with template cache
         if use_cache:
-            body['settings']['buildTargetCopyCache'] = template_target
-            print(f"Using template cache build target: {template_target}")
+           if use_cache:
```

Comment:
This extra nested if probably isn't needed, right?

Reply:
Good catch, no need for double ifs like that